ddp-workspace/projects/ddp-atcp/src/app/components/activityForm/app-atcp-activity-base.component.ts

```diff
@@ -254,6 +257,15 @@ export class AtcpActivityBaseComponent extends ActivityComponent implements OnIn
           }
         })
     );
+    this.anchor.addNew(
+      combineLatest([this.languageService.onLanguageChange(), this.languageService.getParticipantGuidBeforeLanguageChange()])
+      .pipe(delay(2000))
```

Comment:
Wondering if the delay is needed here? 
If yes - why it is 2s (can it less) ?

Reply:
It was only done to allow the backend a little extra time to respond. I adjusted to 150 ms because 2 seconds is undoubtedly too long.